app/services/account_oauth_getter.go

```diff
@@ -0,0 +1,28 @@
+package services
+
+import (
+	"github.com/keratin/authn-server/app/data"
+	"github.com/pkg/errors"
+)
+
+func AccountOauthGetter(accountStore data.AccountStore, accountID int) ([]map[string]interface{}, error) {
```

Comment:
Since this data is returned as part of the account JSON, you might consider implementing it within the existing `AccountGetter`.

Reply:
Nice one! I have updated the code accordingly.